releng/pom.xml

```diff
@@ -26,11 +27,11 @@
 		</repository>
 		<repository>
 			<id>maven</id>
-			<url>http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastStableBuild/artifact/build/maven-repository/</url>
+			<url>${JENKINS_URL}/job/lsp4j/job/master/lastStableBuild/artifact/build/maven-repository</url>
 		</repository>
```

Comment:
is this generated? if yes we would need an additonal pr for that plugin too

Reply:
Oh, good point! I forgot that this kind of poms are generated by the gradle plugin. I will produce a PR for that now.

Reply:
As we figured out the Gradle plugin can't be customized to support this. We decided to drop the usage of p2gen.